game_event_player/tests.py

```diff
@@ -0,0 +1,94 @@
+from django.contrib.auth import get_user_model
+import pytest
+from django.db.utils import IntegrityError
+from django.core.exceptions import ValidationError
+from .models import GameEventPlayer
+from game_event.models import GameEvent
+from player.models import Player, BallGame
+from court.models import Court
+from decimal import Decimal
+from datetime import datetime
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def user():
+    return get_user_model().objects.create_user(
+        username='testuser',
+        password='testpass'
+    )
+
+
+@pytest.fixture
+@pytest.mark.django_db
```

Comment:
i think this is not necessary `@pytest.mark.django_db`

Reply:
> i think this is not necessary `@pytest.mark.django_db`, should only be above the Test Class

The fixtures functions are not inside the class. If you want me to remove the @pytest.mark.django_db
it means the functions should be inside the class. Is that what you want?
After the confitest.py PR is merged we will need to add it all those files meaning fixtures wont be inside of a class.

Reply:
@MaayanMashhadi mark.django_db is used to mark a test function as requiring the database
the fixture is for providing data for tests.

there is no reason to put the fixtures inside the class or under`@pytest.mark.django_db`

try removing it and see what is the outcome

Reply:
done